#! /bin/sh /usr/share/dpatch/dpatch-run
## 02cryptostyle-configurein.dpatch by Julien Delange <julien@gunnm.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fails to identify crypto++ release (#348579)

@DPATCH@
--- ../configure.in.old	2006-03-26 08:18:06.000000000 +0200
+++ ../configure.in	2006-03-26 08:18:37.000000000 +0200
@@ -243,7 +243,7 @@
 
 case "$CRYPTO_PP_STYLE" in
  
- gentoo_debian) CXXFLAGS="$CXXFLAGS -I$crypto_prefix/include -D__CRYPTO_DEBIAN_GENTOO__"
+ gentoo | debian) CXXFLAGS="$CXXFLAGS -I$crypto_prefix/include -D__CRYPTO_DEBIAN_GENTOO__"
                 # Debian uses libcrypto++5.1 - it's not my fault, please soda patch the package
                 LDFLAGS="$LDFLAGS -L$crypto_prefix/lib"
 		CRYPTOLIBS="-lcrypto++"
